plugins/apps/clementine.mk

```diff
@@ -0,0 +1,45 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := clementine
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 1.3
+$(PKG)_CHECKSUM := 55e8d283ed02e3f2376a88b012cd2003e00fef95a98b6a919d67cb57e96b8617
+$(PKG)_SUBDIR   := Clementine-$($(PKG)_VERSION)
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://github.com/$(PKG)-player/$(PKG)/archive/$($(PKG)_VERSION).tar.gz
+$(PKG)_DEPS     := gcc boost chromaprint cryptopp dlfcn-win32 fftw glew gst-plugins-bad \
+                   gst-plugins-good gst-plugins-ugly libarchive libechonest libid3tag \
+                   libmpcdec libplist libusb1 protobuf qtsparkle_qt4 sparsehash
+
+define $(PKG)_UPDATE
+    $(call MXE_GET_GITHUB_TAGS, clementine-player/clementine)
+endef
+
+define $(PKG)_BUILD
+    cd '$(BUILD_DIR)' && '$(TARGET)-cmake' '$(SOURCE_DIR)' \
+        -DCMAKE_INSTALL_PREFIX=$(PREFIX)/$(TARGET)/apps/$(PKG)
+    $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)'
+    $(MAKE) -C '$(BUILD_DIR)' -j 1 install
+
+    $(if $(BUILD_SHARED),
+        $(INSTALL) -d '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
+        $(INSTALL) '$(PREFIX)/$(TARGET)'/bin/{libgstapetag,libgstapp,libgstasf,libgstaudioconvert,libgstaudiofx,libgstaudioparsers,libgstaudioresample,libgstaudiotestsrc,libgstautodetect,libgstcdio,libgstcoreelements,libgstdirectsoundsink,libgstequalizer,libgstfaad,libgstflac,libgstgdp,libgstgio,libgsticydemux,libgstid3demux,libgstisomp4,libgstlame,libgstmad,libgstogg,libgstopus,libgstplayback,libgstreplaygain,libgstsouphttpsrc,libgstspectrum,libgstspeex,libgsttaglib,libgsttcp,libgsttypefindfunctions,libgstudp,libgstvolume,libgstvorbis,libgstwavpack,libgstwavparse}.dll \
+        '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins';
```

Comment:
It is looking really awful:
```
        $(INSTALL) '$(PREFIX)/$(TARGET)'/bin/{libgstapetag$(comma)libgstapp$(comma)libgstasf$(comma)libgstaudioconvert$(comma)libgstaudiofx$(comma)libgstaudioparsers$(comma)libgstaudioresample$(comma)libgstaudiotestsrc$(comma)libgstautodetect$(comma)libgstcdio$(comma)libgstcoreelements$(comma)libgstdirectsoundsink$(comma)libgstequalizer$(comma)libgstfaad$(comma)libgstflac$(comma)libgstgdp$(comma)libgstgio$(comma)libgsticydemux$(comma)libgstid3demux$(comma)libgstisomp4$(comma)libgstlame$(comma)libgstmad$(comma)libgstogg$(comma)libgstopus$(comma)libgstplayback$(comma)libgstreplaygain$(comma)libgstsouphttpsrc$(comma)libgstspectrum$(comma)libgstspeex$(comma)libgsttaglib$(comma)libgsttcp$(comma)libgsttypefindfunctions$(comma)libgstudp$(comma)libgstvolume$(comma)libgstvorbis$(comma)libgstwavpack$(comma)libgstwavparse}.dll \
            '$(PREFIX)/$(TARGET)/apps/$(PKG)/bin/gstreamer-plugins'
```

In my opinion, previous variant is looking much more nice. Do you really insist on this https://github.com/mxe/mxe/pull/1277#discussion_r85070957
?